compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1200,6 +1200,21 @@ LogicalResult WinogradOutputTransformOp::reifyResultShapes(
 // AttentionOp
 //===----------------------------------------------------------------------===//
 
+void AttentionOp::build(::mlir::OpBuilder &odsBuilder,
+                        ::mlir::OperationState &odsState,
+                        ::mlir::TypeRange results, ::mlir::Value query,
+                        ::mlir::Value key, ::mlir::Value value,
+                        ::mlir::Value scale, ::mlir::ValueRange outputs,
+                        ::mlir::ArrayAttr indexing_maps,
+                        std::optional<::mlir::Value> mask) {
+  Value mask_in;
```

Comment:
NIT: can we do `Value mask_in = mask.has_value() ? mask.value() : Value()`?

Reply:
Sounds good